app/controllers/hackathons/submissions_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Hackathons::SubmissionsController < ApplicationController
+  def index
+    @hackathons = Hackathon.not_approved.where applicant: Current.user
+
+    redirect_to new_hackathons_request_path if @hackathons.none?
+  end
+
+  def new
+    @hackathon = Hackathon.new
+  end
+
+  def create
+    @hackathon = Hackathon.new(hackathon_params)
+    if @hackathon.save context: :submit
+      redirect_to hackathons_requests_path, notice: "Your hackathon has been submitted for approval!"
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show
+    @hackathon = Hackathon.not_approved.where(applicant: Current.user).find(params[:id])
+  end
+
+  private
+
+  def hackathon_params
+    params.require(:hackathon).permit(
+      :name,
+      :website,
+      :logo,
+      :banner,
+      :starts_at,
+      :ends_at,
+      :address,
+      :expected_attendees
```

Comment:
Notes from review on zoom:
- We'll want to add tags (as checkboxes on the form)
	- A cool thing about Rails is that form fields that have a name with brackets (e.g. `tags[]`) will be an array in the controller (`params[:tags].class #=> Array`)
	- We could then so something like `tag_as params[:tags]` (although we should whitelist the tags to prevent users from setting random tags by request manipulation)
- The form should ask if the applicant would like swag. If so, reveal the `swag_mailing_address` fields